src/Stratis.Bitcoin.Features.SmartContracts/ReflectionExecutor/Controllers/SmartContractsController.cs

```diff
@@ -103,7 +112,14 @@ public IActionResult GetCode([FromQuery]string address)
             });
         }
 
-        [Route("balance")]
+        /// <summary>
+        /// Gets the balance of a smart contract in Satoshi. This method only works for smart contract addresses. 
+        /// </summary>
+        /// 
+        /// <param name="address">The address of the smart contract to retrieve the balance for.</param>
+        /// 
+        /// <returns>The balance of a smart contract in Satoshi.</returns>
+        [Route("SC-balance")]
```

Comment:
Remove 'SC-' prefix.

Reply:
This does only return contract balances though. Maybe 'contract-balance' is correct.